prove_validator.go

```diff
@@ -15,7 +15,7 @@ type VerifyWithdrawalCredentialsCallParams struct {
 	ValidatorFields       [][]Bytes32     `json:"validatorFields"`
 }
 
-func (epp *EigenPodProofs) ProveValidatorWithdrawalCredentials(oracleBlockHeader *phase0.BeaconBlockHeader, oracleBeaconState *capella.BeaconState, validatorIndices []uint64) (*VerifyWithdrawalCredentialsCallParams, error) {
+func (epp *EigenPodProofs) ProveValidatorWithdrawalCredentials(oracleBlockHeader *phase0.BeaconBlockHeader, oracleBeaconState *deneb.BeaconState, validatorIndices []uint64) (*VerifyWithdrawalCredentialsCallParams, error) {
```

Comment:
No i mean everywhere. If its for deneb, name it deneb. if it's for fork1 name it fork1

Reply:
The other way (easier) is that you name them the same thing and just change argument types. let's not go halfway though

Reply:
hmm not sure about that, its a bit much.  it makes more sense to me to have separate functions for where it matters.  I don't mind having the same name and different types tho, although it might make it a little less explicit